rocky/tools/templatetags/findings_count.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any
+
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def total_findings(data: dict[str, Any]) -> int:
```

Comment:
Just out of curiosity: although this looks like a useful filter, it's currently only being used bye one template to check for emptiness?

Reply:
Yes, it is only used by one template. As @noamblitz said, it's the best fix for now. Changing it in the report.py file would unfortunately not be backward compatible.

Reply:
I've been looking through the code and I only then notices the file report_extra.py. This is a better place for the filter than in a seperate file, so I've moved it to there.

Reply:
I think @ammar92 means this isn't generic enough to warrant a template function, and instead should just be a part of the view / logic that generates the report-data / context-data.
Somewhere in between, this summing could be a python property of the object that we are sending over which would be accessibly from the template, and only executed when needed.

Reply:
You are right @underdarknl that this would be a bit better during generation of the report, but this method makes it backwards compatible with older reports.

Reply:
The object is just a json of which it's unknown whether this part of the report exists